tools/python/util/mobile_helpers/test/test_check_model_can_use_ort_mobile_pkg.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+import onnx
+import pathlib
+import unittest
+
+from testfixtures import LogCapture
+from ..check_model_can_use_ort_mobile_pkg import run_check, run_check_with_model
+
+# example usage from <ort root>/tools/python
+# python -m unittest util/mobile_helpers/test/test_check_model_can_use_ort_mobile_pkg.py
+# NOTE: at least on Windows you must use that as the working directory for all the imports to be happy
+
+script_dir = pathlib.Path(__file__).parent
+ort_root = script_dir.parents[4]
+
+ort_package_build_config_filename = \
+    ort_root / 'tools' / 'ci_build' / 'github' / 'android' / 'mobile_package.required_operators.config'
+
+
+def _create_logger():
+    logger = logging.getLogger('default')
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+class TestMobilePackageModelChecker(unittest.TestCase):
+    def test_supported_model(self):
+        with LogCapture() as log_capture:
+            logger = _create_logger()
+            model_path = ort_root / 'onnxruntime' / 'test' / 'testdata' / 'ort_github_issue_4031.onnx'
+            supported = run_check(model_path, ort_package_build_config_filename, logger)
+            self.assertTrue(supported)
+
+            # print(log_capture)
+            log_capture.check_present(
+                ('default', 'INFO', 'Model should work with the pre-built package.'),
```

Comment:
Side question, do  we have this capability, (capture and verify certain log entries exist) for c++ UT?
This can be very helpful for UT
Something like a textlogsink to store logs in an array of strings?

Reply:
We do for something that uses our logging infrastructure. The logging unit tests use a mock sink to capture and validate log output.

The gap may be whether it's easy to provide that to the LoggingManager instance used in the majority of the unit tests.